src/DatePicker/index.js

```diff
@@ -100,7 +102,6 @@ export default class DatePicker {
 
   open() {
     clearTimeout(this.closeTimeout)
-    this.selectDay(this.state.day, true)
     this.ui.calendar.removeAttribute('style')
     this.ui.selectedDay.focus()
```

Comment:
Right now, when you open the date picker, focus is thrown to the currently selected day (today by default). Does that make sense, or should I let it just start from the top with the next/prev buttons?